cmake/projects/CURL/hunter.cmake

```diff
@@ -27,15 +27,30 @@ hunter_add_version(
 )
 
 
-
-hunter_cmake_args(
-    CURL
-    CMAKE_ARGS
-        BUILD_CURL_TESTS=OFF
-        BUILD_CURL_EXE=OFF
-        CMAKE_USE_OPENSSL=ON
-        CMAKE_USE_LIBSSH2=OFF
- )
+if (ANDROID OR IOS)
+    hunter_cmake_args(
+        CURL
+        CMAKE_ARGS
+            BUILD_CURL_TESTS=OFF
+            BUILD_CURL_EXE=OFF
+            CMAKE_USE_OPENSSL=ON
+            CMAKE_USE_LIBSSH2=OFF
+            HAVE_GLIBC_STRERROR_R=1
+            HAVE_GLIBC_STRERROR_R__TRYRUN_OUTPUT=""
+            HAVE_POSIX_STRERROR_R=0
+            HAVE_POSIX_STRERROR_R__TRYRUN_OUTPUT=""
+            HAVE_POLL_FINE_EXITCODE=0
+     )
+else()
+    hunter_cmake_args(
+        CURL
+        CMAKE_ARGS
+            BUILD_CURL_TESTS=OFF
```

Comment:
This part is common for both configuration. Use separate variable like `_curl_cmake_args`:

```cmake
set(_curl_cmake_args BUILD_CURL_TESTS=OFF BUILD_CURL_EXE=OFF ...)
if(ANDROID OR IOS)
  list(APPEND _curl_cmake_args HAVE_GLIBC_STRERROR_R=1 HAVE_GLIBC_STRERROR_R__TRYRUN_OUTPUT="" ...)
endif()
```